1.9.1 Install script not parsing on MacOS

The install script for ocamlfind doesn't appear to be compatible with zsh.
I've tried running the same script with bash, and it does appear to parse.
```
$ opam install ocamlfind                          
The following actions will be performed:
  ∗ install ocamlfind 1.9.1
  ∗ install topkg     1.0.3
  ∗ install ctypes    0.18.0
===== ∗ 3 =====
Do you want to continue? [Y/n] y

<><> Gathering sources ><><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫 
[ctypes.0.18.0] found in cache
[ocamlfind.1.9.1] found in cache

[topkg.1.0.3] found in cache

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫 
[ERROR] The installation of ocamlfind failed at "make install".

#=== ERROR while installing ocamlfind.1.9.1 ===================================#
# context     2.0.5 | macos/x86_64 | ocaml-base-compiler.4.08.1 | https://opam.ocaml.org#d9074796
# path        ~/.opam/4.10.0/.opam-switch/build/ocamlfind.1.9.1
# command     ~/.opam/opam-init/hooks/sandbox.sh install make install
# exit-code   2
# env-file    ~/.opam/log/ocamlfind-77354-c743ac.env
# output-file ~/.opam/log/ocamlfind-77354-c743ac.out
### output ###
# [...]
# if [ "1" -eq 1 ]; then \
#     for x in camlp4 dbm graphics labltk num ocamlbuild; do \
#       if [ -f "/Users/ishaangandhi/.opam/4.10.0/lib/$x/META" ]; then \
#         if ! grep -Fq '[distributed with Ocaml]' "//Users/ishaangandhi/.opam/4.10.0/lib/$x/META"; then \
#           rm -f site-lib-src/$x/META; \
#         fi \
#       fi \
#     done; \
#     test -f "site-lib-src/num/META" || rm -f "site-lib-src/num-top/META"; \
#   fi
# zsh:1: parse error near `done'
# make: *** [check-installation] Error 1



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫 
┌─ The following actions failed
│ ∗ install ocamlfind 1.9.1
└─ 
╶─ No changes have been performed
```

--- a/Makefile
+++ b/Makefile
@@ -90,8 +90,8 @@ check-installation:
       if [ -f "$(prefix)$(OCAML_SITELIB)/$$x/META" ]; then \
         if ! grep -Fq '[distributed with Ocaml]' "$(prefix)/$(OCAML_SITELIB)/$$x/META"; then \
           rm -f site-lib-src/$$x/META; \
-        fi \
-      fi \
+        fi; \
+      fi; \
     done; \
     test -f "site-lib-src/num/META" || rm -f "site-lib-src/num-top/META"; \
   fi

